rabbitmq-quickstart/rabbitmq-quickstart-producer/src/test/java/org/acme/rabbitmq/producer/QuteResourceIT.java

```diff
@@ -0,0 +1,8 @@
+package org.acme.rabbitmq.producer;
+
+import io.quarkus.test.junit.NativeImageTest;
+
+@NativeImageTest
+public class QuteResourceIT extends QuotesResourceTest {
```

Comment:
Wrong name?

Reply:
You missed the o (I was thinking it was the template engine)

Reply:
🤦‍♂️. Sorry about this.

Reply:
Fixed.